wrappers/python/examples/wnt/__init__.py

```diff
@@ -8,3 +8,10 @@
         Copyright 2018 Wirepas Ltd. All Rights Reserved.
         See file LICENSE.txt for full license details.
```

Comment:
I failed to see the mention to LICENSE.txt which does not exist anymore. It should read LICENSE file instead.

Since this is likely to be in the examples as well, could you start a separate PR to change the license headers accordingly? And while at it, update the year.

Reply:
Ok, I'll create separate PR about it